pages/governance.md

```diff
@@ -0,0 +1,280 @@
+other [core contributors] and the [Steering Council].
+
+### Repository Maintainers
+
+Repository maintainers are trusted members of the community who help the [core
+contributors] by managing a limited number of repositories in the [JuMP-dev
+Github organization].
+
+Any core contributor has individual authority to appoint a community member as a
+repository maintainer of a repository with notification to the other [core
+contributors].
+
+Before becoming a repository maintainer, it is expected that the community
+member will have been an active participant in the development and maintenance
+of a repository for a sustained period of time. This includes triaging issues,
+proposing and reviewing pull requests, and updating any binary dependencies as
+needed.
+
+Community members may be maintainers of multiple repositories at the same time.
+
+Repository maintainers can step back from their role at any time by informing a
+core developer. Furthermore, [core contributors], by consensus, can choose to
+remove repository maintainers for any reason (typically inactivity).
+
+### NumFOCUS
+
+JuMP is a [fiscally sponsored project of NumFOCUS](https://numfocus.org/project/jump),
+a 501(c)(3) public charity in the United States. NumFOCUS provides JuMP with
+fiscal, legal, and administrative support to help ensure the health and
+sustainability of the project.
+
+### Steering Council
+
+The Steering Council supports the [core contributors] by representing JuMP in
+all interactions with [NumFOCUS](#numfocus), e.g., by attending NumFOCUS's
+annual summit. In addition, the Steering Council:
+
+* Approves expenditures related to JuMP and paid through the NumFOCUS account.
+* Negotiates and approves contracts between NumFOCUS and external contractors
+  who provide paid work to JuMP.
+
+The current members of the Steering Council are:
+
+* Miles Lubin, Google (@mlubin)
+* Juan Pablo Vielma, Google (@juan-pablo-vielma)
+* Joey Huchette, Rice University (@joehuchette)
+* Oscar Dowson (@odow)
+* Changhyun Kwon, U. South Florida (@chkwon)
+
+A member of the Steering Council may leave the council by notifying the Steering
+Council and [core contributors]. The remaining members, in consultation with the
+[core contributors], will invite a member of the community to join in order to
+maintain a quorum of five members.
+
+## Decision Making Process
+
+This section outlines how financial and non-financial decisions are made in the
+JuMP project.
+
+### Financial Decisions
+
+All financial decisions are made by the [Steering Council] to ensure any funds
+are spent in a manner that furthers the [mission](#mission) of JuMP. Financial
+decisions require majority approval by [Steering Council] members.
+
+Community members proposing decisions with a financial aspect should contact the
+[Steering Council] directly, or table their proposal as an agenda item for
+discussion on a [monthly developer call](#monthly-developer-call).
```

Comment:
make it clearer how proposals are tabled, same for non-financial.
possibly this additional detail could live in: ### Monthly Developer Call
and could be as simple as ping mlubin or odow